examples/react-app/backend/architect.yml

```diff
@@ -14,26 +14,11 @@ dependencies:
       POSTGRES_PASSWORD: architect
       POSTGRES_DB: test_database
 parameters:
-  POSTGRES_HOST:
-    value_from:
-      dependency: db
-      value: $INTERNAL_HOST
-  POSTGRES_PORT:
-    value_from:
-      dependency: db
-      value: $INTERNAL_PORT
-  POSTGRES_USER:
-    value_from:
-      dependency: db
-      value: $POSTGRES_USER
-  POSTGRES_PASSWORD:
-    value_from:
-      dependency: db
-      value: $POSTGRES_PASSWORD
-  POSTGRES_DB:
-    value_from:
-      dependency: db
-      value: $POSTGRES_DB
+  POSTGRES_HOST: ${ dependencies['db'].interfaces.postgres.internal.host }
```

Comment:
My first note is that every "parameter" here that I've changed (in both frontend and backend) is what we would consider to be a "trait" after [issue43](https://gitlab.com/architect-io/product/-/issues/43). Is that correct?